docs/export.md

```diff
@@ -0,0 +1,71 @@
+# Export
+
+ROBOT can export details about ontology entities as a table (TSV or CSV). At minimum, the `export` command expects an input ontology (`--input`), a set of columns (`--columns`), and a file to write to (`--export`):
+
+```
+robot export --input nucleus.owl \
+  --columns "CURIE LABEL" \
+  --export nucleus.csv
+```
+
+### Columns
+
+The `--columns` are space-separated special keywords or properties used in the ontology. These should always be enclosed in double quotes.
```

Comment:
I disagree with the choice of space separated list for the columns. It makes sense for this argument to exactly match the header row of the output table, which means giving the user the choice between comma- and tab-separated lists. This will work better when using labels.

The keywords for this input should also exactly match the output. They should be consistently singular with spaces, not a mix of singular and plural with hyphens.

Reply:
Since this is exactly matching the header, do you think the option should be called `--header` instead? Or is `--columns` appropriate?

Reply:
Yes, I think `--header` is better.